astropy/units/format/ogip.py

```diff
@@ -355,4 +357,10 @@ def to_string(
                     UnitsWarning,
                 )
 
-        return cls._to_string(unit, fraction=fraction)
+        return super().to_string(unit, fraction=fraction)
+
+    @classmethod
+    def format_exponential_notation(
+        cls, val: float | np.number, format_spec: str = "g"
```

Comment:
I spotted a typing error.
```suggestion
        cls, val: UnitScale | np.number, format_spec: str = "g"
```